js/bkex.js

```diff
@@ -232,52 +254,85 @@ module.exports = class bkex extends Exchange {
          * @method
          * @name bkex#fetchMarkets
          * @description retrieves data on all markets for bkex
+         * @see https://bkexapi.github.io/docs/api_en.htm?shell#basicInformation-1
+         * @see https://bkexapi.github.io/docs/api_en.htm?shell#contract-market-symbols
          * @param {object} params extra parameters specific to the exchange api endpoint
          * @returns {[object]} an array of objects representing market data
          */
-        const response = await this.publicGetCommonSymbols (params);
+        const spotMarkets = await this.publicSpotGetCommonSymbols (params);
         //
-        // {
-        //     "code": "0",
-        //     "data": [
-        //         {
-        //             "minimumOrderSize": "0",
-        //             "minimumTradeVolume": "0E-18",
-        //             "pricePrecision": "11",
-        //             "supportTrade": true,
-        //             "symbol": "COMT_USDT",
-        //             "volumePrecision": 0
-        //         },
-        //     ],
-        //     "msg": "success",
-        //     "status": 0
-        // }
+        //     {
+        //         "code": "0",
+        //         "data": [
+        //             {
+        //                 "minimumOrderSize": "0",
+        //                 "minimumTradeVolume": "0E-18",
+        //                 "pricePrecision": "11",
+        //                 "supportTrade": true,
+        //                 "symbol": "COMT_USDT",
+        //                 "volumePrecision": 0
+        //             },
+        //         ],
+        //         "msg": "success",
+        //         "status": 0
+        //     }
         //
-        const data = this.safeValue (response, 'data', []);
+        const swapMarkets = await this.publicSwapGetMarketSymbols (params);
```

Comment:
@Dan-krm in these cases, we should use `Promise.all()` to avoid waiting for one before requesting the other.